### ISSUE TITLE
Editor: Saving poster image doesnt respect actuall view

### ISSUE THREAD

**Issue by phovorka** [google/model-viewer#3892]
Hi, i am not sure if this is a bug but from user point of view it should be more dumb-resistant.
Description: 
By saving a poster image , i would like to save picture of actual view i see on monitor, But it saves only default view. 

Solution:
Workaround... i have to press "save current as initial" in another tab to have the correct result of that image..

Its just feedback 
Cheers Petr

**Comment by elalish** [comment]
Yeah, that's fair. Interested in doing a PR? It's all in the space-opera package.

**Comment by phovorka** [comment]
ok

**Comment by phovorka** [comment]
im not sure if i can create PR with my GitHub acount ...i dont see simple link to do it..... so i will let on you

**Comment by elalish** [comment]
I'm going to call this intended behavior for now. The reason is the poster is intended to go with the model-viewer element that has been set up, which is why it uses the attribute values for the orientation. If you just want a screenshot, you can use the regular screen-shot facilities of the OS, or click the "save as initial" button. But if anyone has other UX idea, please post them here.
